tor/core/posts.py

```diff
@@ -35,7 +35,8 @@ def check_submissions(subreddit, config):
         if (
             post.domain in config.image_domains or
             post.domain in config.audio_domains or
-            post.domain in config.video_domains
+            post.domain in config.video_domains or
+            subreddit in config.subreddits_domain_filter_bypass
```

Comment:
since we're touching this anyway... What about:

```python
if (post.domain in config.image_domains + config.audio_domains + config.video_domains or
    subreddit in config.subreddits_domain_filter_bypass):
```

Reply:
That would work, but is there a performance reason for it or just clarity? I'd probably just format it the same way as above, so I'm not sure overall clarity would change.

Reply:
Both clarity and a performance reason, as far as I can see. It's negligible as far as the gains of each, but to me it lends to an abstraction of a method like `is_whitelisted_domain()`.

I suppose all of this conditional would be put in there in that case, though. Carry on!